src/document/PathListStore.ts

```diff
@@ -44,18 +44,22 @@ export const PathListStore = types
   })
   .actions((self) => {
     return {
-      setActivePathUUID(uuid: string) {
-        if (self.pathUUIDs.includes(uuid)) {
-          self.activePathUUID = uuid;
-        }
-      },
-      addPath(name: string, select: boolean = false): string {
+      disambiguateName(name: string) {
```

Comment:
![image](https://github.com/SleipnirGroup/Choreo/assets/70111132/add179de-eea2-4f79-9aa5-a33742be4db7)
this does not appear to work properly

Reply:
Works as intended, but my intent is probably non-ideal. I wanted a difference between copying "NewPath" twice vs copying NewPath and then copying that copy.

Reply:
i guess that makes sense, but it does feel a little weird and arbitrary to me without being told that thats the logic behind it

Reply:
I think it's fine as-is, considering that most people won't be spam-clicking it.